### PR TITLE
improved UI

### PR DESCRIPTION
**Pull request by tend2infinity**
## Related Issue 

I have made the placeholder at the filter region completely visbile.

Closes: #138

### Describe the changes you've made
Just a few inline style additions in the filter.jsx

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement in the existing features
- [ ] Other

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Screenshot
guruji-bug1

### CODE REVIEW

**Review by Namanl2001 — APPROVED** [review]
Have you tested your changes on mobile phone?

**Comment by tend2infinity** [comment]
@Namanl2001 yes it works alright!

File: src/components/filter.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ class filter extends Component {
         >
           <Form inline my='2 lg-0' onSubmit={this.handleSubmit}>
             <Row>
-              <Col>
+              <Col style={{ paddingRight: '50px', paddingLeft: '45px' }}>
```
Comment (Namanl2001) [inline src/components/filter.jsx:36]: Would be nice if we can move these inline styles to "styles.css". WDYT?

**Comment by tend2infinity** [comment]
Actually I didn't see that there is already a style.css file available in the components folder, I can do it.

**Comment by tend2infinity** [comment]
Ok I've done that

File: src/components/filter.jsx (hunk 1 above)
Comment (Namanl2001) [reply src/components/filter.jsx]: I guess you forget to add the className here? 🤔

**Comment by sagar-barapatre** [comment]
#136 is also doing the same thing. In that PR too the placeholder is completely visible. For same things two PR's are made @Namanl2001

**Comment by tend2infinity** [comment]
Sorry I've been inactive for a while due to some personal issues, I'll update my code right now

**Review by Namanl2001 — APPROVED** [review]
LGTM! From next time please remember to make a new branch for new changes. Thanks

**Pull request merged**